src/cmd/services/m3coordinator/main/main.go

```diff
@@ -51,6 +52,8 @@ func main() {
 	}
 
 	server.Run(server.RunOptions{
-		Config: cfg,
+		Config:             cfg,
+		QueryRouter:        httpd.NewQueryRouter(),
+		InstantQueryRouter: httpd.NewQueryRouter(),
```

Comment:
Can we have these nil and then default to `httpd.NewQueryRouter` if they haven't been overwritten instead?